app/Providers/PhoenixServiceProvider.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Rogue\Providers;
+
+use Rogue\Services\Phoenix\Phoenix;
+use Illuminate\Support\ServiceProvider;
+
+class PhoenixServiceProvider extends ServiceProvider
```

Comment:
Since you're not injecting any custom dependencies or config values, this shouldn't be necessary. Laravel can resolve the `\Rogue\Services\Phoenix` without it being explicitly registered in a service provider.